Improve wording about credential expiration, revocation, and destruct… (#633)
usnistgov/FIPS201: _FIPS201/requirements.md

    for identity proofing and whose fingerprints are checked against databases is the person to whom the
    credential is issued.
+ No credential is issued unless requested by the proper authority.

> + A credential remains serviceable only up to its expiration date and a process
>     exists to invalidate, revoke, or destroy any derived PIV credentials associated with an expired, revoked, or destroyed PIV Card.

Reviewer: ```suggestion
+ A credential remains serviceable only up to its expiration date and a process 
    exists that invalidates revokes, or destroys  credentials.
```

Not sure if my alternate text needs further refinement.   I thought that the text should be about 'credentials' in general here (as it is in all other bulleted items) - hence I was trying to avoid going into Derived PIV/vs PIV Card details.  Also, Derived PIV's don't have to be revoked, invalidated or destroyed, when PIV card is destroyed.  Derived PIV can provide a way to continue authentication when PIV card is replaced.
